FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpMsg.kt

```diff
@@ -294,10 +287,28 @@ object PgpMsg {
           message = "There is no suitable decryption key"
         )
       }
+      // other PGP error, fallback to default error return in the bottom
+      exception = ex
+    } catch (ex: Exception) {
+      if (
+        ex is IOException &&
+        ex.message?.contains("crc check failed in armored message") == true
```

Comment:
Does `PGPainless` return exception as `IOException`? it would be better to have a predefined exception here. For now, it looks a little ugly